src/FakeItEasy/ICallOrderingContext.cs

```diff
@@ -0,0 +1,20 @@
+﻿namespace FakeItEasy
+{
+    using System;
+    using FakeItEasy.Core;
+
+    /// <summary>
+    /// Represents a context to make assertions on the order of the calls made on a fake.
+    /// </summary>
+    public interface ICallOrderingContext
+    {
+        /// <summary>
+        /// Checks that the next call was made in order.
+        /// </summary>
+        /// <param name="fakeManager">The manager for the fake on which the call was made.</param>
+        /// <param name="callPredicate">The predicate used to match the call.</param>
+        /// <param name="callDescription">The description of the call.</param>
+        /// <param name="repeatConstraint">The repeat constraint for the call.</param>
+        void CheckNextCall(FakeManager fakeManager, Func<IFakeObjectCall, bool> callPredicate, string callDescription, Repeated repeatConstraint);
+    }
+}

```

Comment:
I thought about that, but I wasn't too happy about exposing the class directly... But I guess it's better than exposing something that's meaningless to the user.